### ISSUE TITLE
GetTweetsFromUserIdAsync with TweetOptions.Attachments: null reference exception

### ISSUE THREAD

**Issue by Foxite** [Xwilarg/TwitterSharp#3]
```csharp
await twitter.GetTweetsFromUserIdAsync(user.Id, new[] { TweetOption.Attachments }, Array.Empty<UserOption>(), Array.Empty<MediaOption>())
```

```
Unhandled exception. System.NullReferenceException: Object reference not set to an instance of an object.
   at TwitterSharp.Response.RTweet.Tweet.TwitterSharp.Model.IHaveMedia.GetMedia()
   at TwitterSharp.Client.TwitterClient.IncludesParseMedias(IHaveMedia data, Includes includes)
   at TwitterSharp.Client.TwitterClient.IncludesParseMedias(IHaveMedia[] data, Includes includes)
   at TwitterSharp.Client.TwitterClient.InternalIncludesParse[T](Answer`1 answer)
   at TwitterSharp.Client.TwitterClient.ParseArrayData[T](String json)
   at TwitterSharp.Client.TwitterClient.GetTweetsFromUserIdAsync(String userId, TweetOption[] tweetOptions, UserOption[] userOptions, MediaOption[] mediaOptions)
```

I think this happens when not all, but some of the returned tweets have attachments. This means that [this check](https://github.com/Xwilarg/TwitterSharp/blob/master/TwitterSharp/Client/TwitterClient.cs#L82) will pass, but when it gets [here](https://github.com/Xwilarg/TwitterSharp/blob/ecb5762d7d3b81c8682fc90845585bcea38573c3/TwitterSharp/Client/TwitterClient.cs#L53) on the first tweet that does not have attachments, then an NRE will occur because the Attachments object is null.

This problem happens right now for [twitter.com/rivm](https://twitter.com/rivm). Though the nature of the problem means that it may no longer happen in a few hours.

**Comment by Xwilarg** [comment]
I'll have a look at that and I'll keep you updated, thanks for the bug report!

**Comment by Xwilarg** [comment]
Okay it should be fixed with the version 1.0.1

**Comment by Xwilarg** [comment]
As for the other issue, I'm closing this after 2 days of inactivity since the problem is solved.

Feel free to open a new one if needed
